.gitignore

```diff
@@ -1 +1,8 @@
 dist
+node_modules
```

Comment:
I'm curious how a `node_modules` directory gets created in the root directory? For `webapp` is already in a [`.gitignore`](https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/webapp/.gitignore#L1)

Reply:
This happens to me when I accidentally add a new dependency using `npm install` in the root dir, rather than the `webapp` dir. I agree that it doesn't make much sense to have it in the root dir's `.gitignore`, since no dependencies should be in the root dir.

Reply:
Given that `node_modules` should not be in the root dir, having it in  `.gitignore` could conceal it's existence and confuse devs.

@marianunez Thoughts on removing `node_modules` from `.gitignore`?

Reply:
You're right @mickmister I may have installed outside of the `webapp` dir. Thanks for catching this @hanzei!

Agreed it does not make sense here. Will open a PR to remove it.